docs/content/en/docs/concepts/architecture/components/scheduler/_index.md

```diff
@@ -14,3 +14,152 @@ The Lifecycle Scheduler can trigger events such as
 deployment, testing, and remediation at specific times or intervals.
 The Keptn Scheduler guarantees that Pods are initiated only after
 the Pre-Deployment checks are completed.
+
+## Architecture of Keptn Scheduler
+
+To integrate the `Keptn Lifecycle Scheduler` into a Kubernetes cluster,
+a Kubernetes manifest is applied with specific **Keptn annotations**.
+These annotations identify the workload and specify the
+pre-deployment and post-deployment tasks and evaluations
+associated with it.
+
+## How it works
+
+The `Keptn Scheduler`, integrated into the `Keptn Lifecycle Operator`,
+which is equipped with multiple controllers for managing
+**Keptn Custom Resource Definitions (CRDs)** and a powerful
```

Comment:
```suggestion
**Keptn resources** and a powerful
```
To clarify the terminology, a CRD (Custom Resource Definition) yaml file/manifest describes the structure of our resources (KeptnApp/KeptnWorkload, etc...), while a CR manifest describes a instances of those resources.  We also have Kubernetes resources like Deployment, Namespace, etc.  We are gravitating to just calling these "resources" in text for the most part.